README.md

```diff
@@ -145,15 +131,60 @@ cat <<EOF >lst.xml
   <lib name="sdl"/>
 </tool>
 EOF
-compile the TrackLooper repository if you haven’t
 scram setup lst.xml
 cmsenv
-git cms-checkdeps -a
 scram b -j 12
-cd RecoTracker/LST/test/
-cmsRun LSTAlpakaTester.py
 ```
 
+### Run the LST reconstruction in CMSSW
+A simple test configuration of the LST reconstruction can be run with the command:
+```bash
+cmsRun RecoTracker/LST/test/LSTAlpakaTester.py
+```
+
+For a more complete workflow, one can run a modified version of the 21034.1 workflow.
+To get the commands of this workflow, one can run:
+```bash
+runTheMatrix.py -w upgrade -n -e -l 21034.1
+```
+
+For convenience, the workflow has been run for 100 events and the output is stored here:
+```bash
+/ceph/cms/store/user/evourlio/LST/step2_21034.1_100Events.root
+```
+
+For enabling the LST reconstruction in the CMSSW tracking workflow, a modified step3 needs to be run.
+This is based on the step3 command of the 21034.1 workflow with the following changes:
+   - Use dummy PU input files by changing the argument of the `--pileup_input` flag to `file:file.root` (this will be fixed manually in the configuration file)
+   - Add at the end of the command: `--procModifiers gpu,trackingLST,trackingIters01 --no_exec`
```

Comment:
I'm not sure why this happens at this step, but I think it's the same issue I cannot find a "MeasurementDet"?
ModuleNotFoundError: No module named 'RecoTracker.MeasurementDet.measurementTrackerEventDefault_cfi'